src/rust/uk.mangainua/src/helper.rs

```diff
@@ -0,0 +1,98 @@
+use aidoku::
+{
+	std::String,
+	MangaContentRating
+};
+
+pub fn is_nsfw(genre: String) -> MangaContentRating{
+
+	let nsfwcategories: String = String::from("Еччі Юрі Яой"); // maybe new
+	if nsfwcategories.contains(&genre)
+	{
+		return MangaContentRating::Nsfw;
+	}
+	//return MangaContentRating::Safe;
+	MangaContentRating::Safe
```

Comment:
you're allocating an extra string, avoid doing `String::new` when it's not necessary
```suggestion
	let nsfwcategories = "Еччі Юрі Яой"; // maybe new
	if nsfwcategories.contains(&genre) {
		MangaContentRating::Nsfw
	} else {
		MangaContentRating::Safe
	}
```